go.mod

```
@@ -1,30 +1,29 @@
 module helm.sh/chartmuseum
 
-go 1.21
+go 1.23
```

I had to set this to build 🤔 
```suggestion
go 1.23.0
```

or 

```suggestion
go 1.23

toolchain go1.23.0
```